tests/python/sdk/common.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import io
+import zipfile
+from pathlib import Path
+
+from cvat_sdk.core.proxies.jobs import Job
+from cvat_sdk.core.proxies.projects import Project
+from cvat_sdk.core.proxies.tasks import Task
+from cvat_sdk.core.proxies.types import Location
+
+from shared.utils.config import IMPORT_EXPORT_BUCKET_ID
+from shared.utils.s3 import S3Client
+from shared.utils.s3 import make_client as make_s3_client
+
+from .util import make_pbar
+
+ProjectOrTaskOrJob = Project | Task | Job
+
+
+class _TestDatasetExport:
```

Comment:
Could you rename this class so that it has no underscore prefix? It's imported in other files, so it should be considered public.